docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Build a Mobile App for Token Staking on Fuse.
+sidebar_position: 16
+---
+
+# Build a Mobile App for Token Staking on Fuse.
+In this guide, we will walk through the steps to build a mobile app using the Fuse Flutter SDK method [Token Staking](https://pub.dev/packages/fuse_wallet_sdk#staking). This guide will walk you through setting up your Flutter environment, initializing the Fuse SDK, and implementing wallet creation and token staking functionalities. Let’s dive right in!
+
+## Pre-requites:
+
+- Basic knowledge of Dart/Flutter
+- An API Key from the [Console](https://console.fuse.io/build)
+
+## Set Up Your Flutter Environment
+First things first, you need to have Flutter installed on your machine. If you haven’t installed it yet, you can find the installation guide on the Flutter official website. Once you have Flutter set up, create a new project by running:
+
+```bash
+flutter create fuse_wallet_app
+cd fuse_wallet_app
+```
+
+Next, add the Fuse Wallet SDK to your project. Open your pubspec.yaml and add the following line under dependencies:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  fuse_wallet_sdk: latest_version
```

Comment:
This doesnt worked for me, replace it with actual latest version

Reply:
Fixed.